# Message describing the sensor joint stiffnesses
Header header
float32[] stiffnesses